source/configuration/modules/omfile.rst

```diff
@@ -1,434 +1,441 @@
 
-.. function:: dirGroupNum [gid]
+=========  ============================================
+dirGroup:  "groupname"
+Default:   current user (running Rsyslog) primary group 
+=========  ============================================
 
-   *Default: process user's primary group*
+Sets the default directroy group name to be used for an action if no explicit one is specified.
 
-   Sets the default dirGroupNum to be used for an action if no
-   explicit one is specified.
- 
+============  ============================================
+dirGroupNum:  "gid (number)"
+Default:      current user (running Rsyslog) primary group 
+============  ============================================
 
+Sets the default directory group (*gid*) to be used for an action if no explicit one is specified.
+   
 Action Parameters
-^^^^^^^^^^^^^^^^^
-Note that **one** of the parameters *file* or *dynaFile* must be specified. This
-selects whether a static or dynamic file (name) shall be written to.
-
-.. function::  file [fileName]
-
-   *Default: none*
-
-   This creates a static file output, always writing into the same file.
-   If the file already exists, new data is appended to it. Existing
-   data is not truncated. If the file does not already exist, it is
-   created. Files are kept open as long as rsyslogd is active. This
-   conflicts with external log file rotation. In order to close a file
-   after rotation, send rsyslogd a HUP signal after the file has been
-   rotated away.
-
-.. function::  dynaFile [templateName]
+=================
 
-   *Default: none*
+Note that **one** of the parameters *file* or *dynaFile* must be specified.
 
-   For each message, the file name is generated based on the given
-   template. Then, this file is opened. As with the *file* property,
-   data is appended if the file already exists. If the file does not
-   exist, a new file is created. The template given in "templateName"
-   is just a regular :doc:`rsyslog template <../templates>`, so all
-   you have full control over how to format the file name.
-   
-   A cache of recent files is kept. Note
-   that this cache can consume quite some memory (especially if large
-   buffer sizes are used). Files are kept open as long as they stay
-   inside the cache.
-   Files are removed from the cache when a HUP signal is sent, the
-   *closeTimeout* occurs, or the cache runs out of space, in which case
-   the least recently used entry is evicted.
+========  ===============
+file:     "/path/to/file"
+Default:  none
+========  ===============
 
-.. function::  template [templateName]
+This parameters set in which file will Rsyslog write. It will always write into the same file.
 
-   *Default: template set via "template" module parameter*
+* If the file does not already exist, it is created.
+* If the file already exists, new data is appended to it.
 
-   Sets the template to be used for this action.
+**Important:** Files are kept open as long as rsyslogd is active.
```

Comment:
The file is opened the first time that rsyslog has a log message to write out, and remains open until rsyslog exits or receives a HUP